src/browser/extension/options/options.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
As we discussed, moved the options component into a separate file. (Probably it’s better to capitalize the file name, but I can’t change this as I’m using Windows with case-insensitive file system.)